### DISCUSSION TITLE
Add customisable language display names

### DISCUSSION THREAD

**Discussion by Nixinova** [lowlighter/metrics#343] [Ideas]
The colours of languages can already be customized, and it would be useful to allow the names themselves to be changed as well. For instance I label all my [NovaSheets](https://github.com/NovaSheets/NovaSheets) code as Less as that is the most similar language but it's a bit misleading to show that 7% of my code is Less, a language I dont use.

E.g.: `plugin_languages_names: "Less:NovaSheets, JavaScript:React"`

**Comment by lowlighter** [comment]
Yeah it shouldn't be hard to implement

**Reply by lowlighter** [reply]
Shipped in 94539fa 🎉
